### ISSUE TITLE
[Site Request] Mangadon

### ISSUE THREAD

**Issue by Kiwi-ma** [manga-download/hakuneko#6038]
Name of the website
Mangadon

Website urls (examples below)
Site: https://mangadon.me/
Manga List: https://mangadon.me/catalogs/show?id=8
Manga example: https://mangadon.me/catalogs/show?id=8&tb=353
Chapter online viewer example: https://mangadon.me/comics/episodes/show?comic_id=353&id=1896

Languages
French

*Website relationship
If applicable describe the relation with any other website (eg : "alternative domain, copy of ...").

Additional details
Add any other context details that you may have found (like template to reuse)

**Comment by MikeZeDev** [comment]
Simple API and scrambled pictures

**Comment by Kiwi-ma** [comment]
Va-t'il être ajouté ?

**Comment by Mahora2106** [comment]
Hello, I would also be interested, do you think you will do it?

**Comment by Mahora2106** [comment]
@MikeZeDev Hello, will the site also arrive in the Hakuneko application?

**Comment by MikeZeDev** [comment]
No. No more websites on Haku. Haru is the new deal, But not finished

**Comment by Mahora2106** [comment]
@MikeZeDev So we can only use it on Haru? However, I don't really understand how it works.

**Comment by MikeZeDev** [comment]
dead
